Framework/test/testTrendingTask.cxx

```diff
@@ -0,0 +1,33 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file    testTrendingTask.cxx
+/// \author  Piotr Konopka
+///
+
+//#include "getTestDataDirectory.h"
+#include "QualityControl/PostProcessingInterface.h"
+//#include "QualityControl/TrendingTask.h"
+#include "QualityControl/PostProcessingFactory.h"
+
+#define BOOST_TEST_MODULE PostProcessingRunner test
+#define BOOST_TEST_MAIN
+#define BOOST_TEST_DYN_LINK
+
+#include <boost/test/unit_test.hpp>
+
+using namespace o2::quality_control::postprocessing;
+
+BOOST_AUTO_TEST_CASE(test_factory)
+{
+
+  //  std::string configFilePath = std::string("json://") + getTestDataDirectory() + "testSharedConfig.json";
```

Comment:
test is empty ?

Reply:
Ah yes, this is a leftover that I should have erased. I am not sure how I could approach testing this without using a database (I guess we don't want to do it, because such tests tend to fail for too many random reasons).

Reply:
indeed, on the other hand in testCcdbDatabase the connection is first tested and only then the test executed if I am not mistaken. Moreover, we use a directory named after the PID to avoid clashes between tests. Maybe this is something that can be done in a different PR, but I would not rule it out. In last resort, the test can be marked in CMake not to be ran in GH.

Reply:
OK, I will write a test. Actually I forgot to write a test for Reductors anyway, this is needed too.